Optik/Classes/ImageViewController.swift

```diff
@@ -159,28 +160,32 @@ internal final class ImageViewController: UIViewController {
             
             self.activityIndicatorView = activityIndicatorView
         }
-        
+
         if let button = likeButton {
             button.translatesAutoresizingMaskIntoConstraints = false
             button.imageView?.contentMode = .scaleAspectFill
             
+           let xAnchorAttribute = likeButtonPosition?.xAnchorAttribute()
+           let yAnchorAttribute = likeButtonPosition?.yAnchorAttribute()
+            
+            
             view.addSubview(button)
             
             view.addConstraint(
                 NSLayoutConstraint(item: button,
-                                   attribute: .bottom,
+                                   attribute: yAnchorAttribute!,
```

Comment:
I added a default value in the init so we always have a value here that's the why I added a exclamation mark here. But I can use a `if let` or `guard let`